cpp/src/io/parquet/chunk_dict.cu

```diff
@@ -92,8 +103,7 @@ struct map_find_fn {
 
 template <int block_size>
 __global__ void __launch_bounds__(block_size, 1)
```

Comment:
`__launch_bounds__(block_size, 1)` indicates only 1 CTA is expected to fit per SM. Is it really intentional to have only 1 CTA per SM?

Reply:
No. Please change.

Reply:
I'd suggest just changing this to `__launch_bounds(block_size)__`, i.e., don't provide an argument for the min number of CTAs.

Reply:
Updates: min number removed. No performance impact.